applications/stepswitch/src/stepswitch_bridge.erl

```diff
@@ -691,3 +692,16 @@ send_deny_emergency_response(OffnetReq, ControlQ) ->
 get_event_type(CallEvt) ->
     {Cat, Name} = kz_util:get_event_type(CallEvt),
     {Cat, Name, kz_call_event:call_id(CallEvt)}.
+
+
+%% If it is an emergency call (i.e 911 call) don't honor `privacy' settings.
+-spec avoid_privacy_if_emergency_call(boolean(), stepswitch_resources:endpoints()) -> stepswitch_resources:endpoints().
+avoid_privacy_if_emergency_call('true', Endpoints) ->
+    Values = [{?KEY_PRIVACY_HIDE_NAME, 'false'}
```

Comment:
Definitely a `lager:info` would be useful here to note the explicit setting of no privacy on the call.